File: Modul-1/C/PRAK110-2310817320007-ZahraNabila.c
#include <stdio.h>
#include <math.h>

int main()
{

printf("Diketahui : \n");

    int a, t, sisiA, sisiB, sisiC;

printf("Alas = ");
scanf("%d", &a);

printf("Tinggi = ");
scanf("%d", &t);

sisiA = t;
sisiB = sqrt(pow(a, 2) + pow(t, 2));
sisiC = a;

    int KelilingSegitiga = sisiA + sisiB + sisiC;
    int LuasSegitiga = 0.5 * a * t;

printf("Jawab : \n");

printf("Sisi A = %d cm\n", sisiA);
printf("Sisi B = %d cm\n", sisiB);
printf("Sisi C = %d cm\n", sisiC);

printf("Keliling = %d cm\n", KelilingSegitiga);
printf("Luas = %d cm\n", LuasSegitiga);

    return 0;
}

File: Modul-1/Python/PRAK110-2310817320007-ZahraNabila.py
import math
print("Diketahui : ")

a = int(input("Alas = "))

t = int(input("Tinggi = "))

sisiA = t
sisiB = int(math.sqrt(a**2 + t**2))
sisiC = a

KelilingSegitiga = sisiA + sisiB + sisiC
LuasSegitiga = 0.5 * a * t

print(f"Jawab : ")

print(f"Sisi A =  {sisiA} cm")
print(f"Sisi B =  {sisiB} cm")
print(f"Sisi C =  {sisiC} cm")

print(f"Keliling =  {KelilingSegitiga} cm")
print(f"Luas =  {LuasSegitiga} cm")